api/http-request-practice.md

```diff
@@ -1,90 +1,246 @@
-# Example Topic Outline (Replace with Title)
+# HTTP Request Practice
 
 ### Projected Time
 
-Example: 30-45 minutes
+** TODO: add after someone dry-runs it**
+Example: ????
 - Lesson: ? min
 - Guided Practice: ?-? min
 - Independent Practice: ? min
 - Check for Understanding: ?-? min
 
 ### Prerequisites
 
-Here are links to lessons that should be completed before this lesson:
-
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
+* [APIs & JSON](./apis-and-json.md)
+* [Twilio API](./twilio-api.md)
+* [REST API](./REST-API.md)
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+HTTP is the most common way you will retrieve anything form the internet: usually web pages or API data (see [API & JSON lesson](https://github.com/Techtonica/curriculum/blob/master/api/apis-and-json.md)). It's the backbone of the web. Since HTTP is so simple and versatile (it's just text!), you can make requests lots of different ways.
```

Comment:
```suggestion
HTTP is the most common way you will retrieve anything from the internet: usually web pages or API data (see [API & JSON lesson](https://github.com/Techtonica/curriculum/blob/master/api/apis-and-json.md)). It's the backbone of the web. Since HTTP is so simple and versatile (it's just text!), you can make requests lots of different ways.
```